client/web/compose/src/store/record.js

```diff
@@ -46,7 +46,7 @@ export default function (ComposeAPI) {
 
         // exclude existing & make unique
         const existing = new Set(getters.set.map(({ recordID }) => recordID))
-        recordIDs = [...new Set(recordIDs.filter(recordID => !existing.has(recordID)))]
+        recordIDs = [...new Set(recordIDs)]
```

Comment:
This will break record resolution(the whole point is we don't refetch the ones we don't have to).
You'll have to remove the updated resolution by hand for the updated records.